Local.md

```diff
@@ -1,94 +1,121 @@
-* Google Docs
-* Google Slides
-* Google Sheets
-* Dropbox Paper
-* Figma
-* Miro
-* Jira
-* Linear
-* GitHub
-
+## Initial Setup ☁️
+* Setup Mac with your Apple account
+* Enable FindMyMac
+* Hardware encryption with backup key stored in iCloud
 
-### Mobile Apps to Install
-* Work Calendar 
-* Work Email
-* Authenticator app for 2FA
+|Name|Details|
+|--|--|
+|[Safari](https://www.apple.com/safari/)|Faster speed and more privacy that alternatives| 
+|[RayCast](https://raycast.com)| Powerful alternative to Spotlight|
+|[Endpoint Management](https://go.jamfnow.com/)|Enroll the computer to our company device management system.|
+
+
+## Terminal :computer:
+
+A useful tutorial: https://towardsdatascience.com/the-ultimate-guide-to-your-terminal-makeover-e11f9b87ac99
+
+|Name|Details|
+|--|--|
+|[iTerm2](https://iterm2.com)|Native terminal replacement that allows you do much more| 
+|[Zsh](https://github.com/ohmyzsh/ohmyzsh/wiki/Installing-ZSH)|Command line shell|
+|[Oh-My-Zsh](https://github.com/ohmyzsh/ohmyzsh)|Allows you add amazing plugins and themes to zsh|
+
+
+#### :electric_plug:  [**Plugins**](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins)
+After you set up Oh-My-Zsh, you will have to edit the hidden file './zshrc' to change themes and add useful plugins. Type 'open ./zshrc' in your terminal to find and edit it. These are the plugins to install.
+
+* `git`
+* `Ruby`
+* `Rails`
+* `autojump`
+
+## Development Environment 
+
+|Name|Details
+|--|--|
+|[Xcode](https://developer.apple.com/xcode/)|| 
+|[Ruby](https://www.ruby-lang.org/en/)|A dynamic, open source programming language with a focus on simplicity and productivity|
+|[Rails](https://rubyonrails.org)| A web application development framework for Ruby|
+|[Redis](https://redis.io/download)|Used for cache, jobs queue|
+|[Postgres](https://postgresapp.com)|Our primary SQL database for record storage|
+
+## Programming tools  🛠
+|Name|Details|
+|--|--|
+|[Github Desktop](https://desktop.github.com)|Easy GUI for using git without terminal commands| 
+|[Postico](https://eggerapps.at/postico/)|GUI for Postgres databases|
+|[Chrome](https://www.google.com/intl/en/chrome/)|Used in development|
+
+## Code Editor
+Any of these code editors are recommended.
+* [Atom](https://atom.io)
+* [SublimeText ](https://www.sublimetext.com) :heavy_dollar_sign:
+* [VSCode](https://code.visualstudio.com)
+
+#### :electric_plug:Plugins
+
+* `Tabnine` - An AI tool that predicts what you're going to type!
+* `Rubocop autocorrect` - A code analyser and format checker to make sure your code look good
+
+## Communication :telephone_receiver:
+|Name|Details|
+|--|--|
+|[Slack](https://slack.com/intl/en-ie/)|The main way to communicate with the team| 
+|[Apple calendar](https://apps.apple.com/us/app/calendar/id1108185179)| To have all your meetings and engagements in one place|
+|[Fantastical](https://apps.apple.com/us/app/fantastical-calendar-tasks/id718043190) :heavy_dollar_sign:|A calendar alternative|
```

Comment:
you can possibly add this to the link above as "Apple Calendar / Fantastical" while linking the fantastical text